.github/workflows/ci.yml

```diff
@@ -84,14 +80,13 @@ jobs:
         -DAVIF_CODEC_RAV1E=ON -DAVIF_LOCAL_RAV1E=ON
         -DAVIF_CODEC_SVT=ON -DAVIF_LOCAL_SVT=ON
         -DAVIF_CODEC_LIBGAV1=ON -DAVIF_LOCAL_LIBGAV1=ON
-        -DAVIF_LOCAL_LIBYUV=ON -DAVIF_LOCAL_JPEG=ON
+        -DAVIF_LOCAL_LIBYUV=ON
         -DAVIF_LOCAL_LIBSHARPYUV=ON
         -DAVIF_BUILD_EXAMPLES=ON -DAVIF_BUILD_APPS=ON
         -DAVIF_BUILD_TESTS=ON -DAVIF_ENABLE_GTEST=ON -DAVIF_LOCAL_GTEST=ON
     - name: Build libavif (ninja)
       working-directory: ./build
       run: ninja
-    - name: Run AVIF Tests (on Linux)
-      if: runner.os == 'Linux'
+    - name: Run AVIF Tests
       working-directory: ./build
-      run: ctest -j $(nproc)
+      run: ctest -j 2
```

Comment:
Why not using all available cores?

Reply:
I mentioned this in the commit message, but neglected to highlight it here. We can't pass `$(nproc)` to the `-j` option of the `ctest` command because the `nproc` command is not available on macOS.

My experiments showed that `nproc` returns 2 in the virtual machine that runs the ubuntu-latest os image, and the `NUMBER_OF_PROCESSORS` environment variable has the value 2 in the virtual machine that runs the windows-latest os image. This is why I pass 2.

It is possible to work harder to get the number of processor cores on macOS, but I didn't look into that in this pull request because it's more important to start running tests.

Reply:
Thank you for the explanation.

Reply:
If we need to use `nproc` on Linux and a different command on macOS, what is the best way to handle this? One idea I have is to have a shell script that invokes a platform-specific command to return the number of processor cores, and then invoke that shell script in place of `$(nproc)`. Do you think this is a good solution?

Reply:
I do not think this is important enough to spend too much time on it, especially if it is just for going from 2 to 3 threads on MacOS only. Anyway having platform-specific commands seems to be the way to go, either in a shell script as you mentioned, or [directly as a GitHub action](https://github.com/orgs/community/discussions/25057) (maybe this is what you meant?).